code/game/objects/items/flamethrower.dm

```diff
@@ -197,12 +198,12 @@
 			attack_self(M)
 
 
-/obj/item/flamethrower/proc/default_ignite(turf/target, release_amount = 0.05)
+/obj/item/flamethrower/proc/default_ignite(turf/target, release_amount = gas_release_amount)
 	//TODO: DEFERRED Consider checking to make sure tank pressure is high enough before doing this...
 	//Transfer 5% of current tank air contents to turf
 	var/datum/gas_mixture/air_transfer = ptank.air_contents.remove_ratio(release_amount)
 	if(air_transfer.gases[/datum/gas/plasma])
-		air_transfer.gases[/datum/gas/plasma][MOLES] *= 5
+		air_transfer.gases[/datum/gas/plasma][MOLES] *= (gas_release_amount * 100)
```

Comment:
This seems bit weird to tie directly since you might want better throughput but not the magic plasma multiplication, could you separate the two values please ?

Reply:
I'd be more than happy to do so.  Although I'm confused as to why.
The multiplication simply serves to release the proportional moles of gas, relative to what was originally in place.

From my understanding, if we increased the release_amount, the air_transfer.gases should also be increased proportionally?

Say if I separated the values, I would have another value called plasma_transfer_quantity, of which I would assign gas_release_amount and then multiply by 100 to achieve the correct value.

If that's all you want me to do, for the purpose of making the code look a bit nicer that's fine.
But I'm not sure what you mean when you say 'magic plasma multiplication'.

Reply:
This code increases the plasma content of the gas sample by five. There's no reason for it except making the burn better, it's not actually relative to the content of the tank hence the "magic". 

Let's say you have tank with 100 moles of plasma in it, if you pour it out all using flamethrower you'll end up with 500 moles of plasma.

Reply:
This could be trivially changed by just removing the increase from the tank of course, but that makes the flamethrower run out very fast.

Reply:
Alright, I see how this is a problem. What I don't see however is the best solution.
air_transfer is already proportional to the amount of remove_ratio.
And in order to get the flamethrower to be effective, the original code multiplied the amount of plasma produced by 5.

This complicates the problem a bit, ideally the syndicate flamethrower should be better than the standard one. Unfortunately it is already limited by the tank size. 
I have not looked at anything regarding gases or tanks before, so I'm not sure what the best solution would be here.

I'd really rather not multiply the amount of plasma by 5, or whatever number really. Needs to be the exact same amount released, for sanity purposes.
Ideally, the syndicate flamethrower should have a larger tank containing the reagents of the operator backpack flamethrower, but I'm not overly sure how to achieve that. 

At this point I'm thinking of redirecting my efforts towards the backpack flamethrower. The tank has to be larger in order to be more effective, and thus must be worn on the back. Makes sense.

Reply:
Anything happening with this ?

Reply:
Nah, didn't want to mess around making bigger tanks, or try to figure out what's wrong with the backpack flamethrower.